resources/views/components/translatable-input.blade.php

```diff
@@ -10,6 +10,10 @@
                 @if ($hint)
                     <x-hearth-hint :for="$name">{{ $hint }}</x-hearth-hint>
                 @endif
+                @isset($interpretationName)
+                    <x-interpretation class="interpretation--start" name="{{ __($interpretationName, [], 'en') }}"
+                        namespace="{{ $interpretationNameSpace }}" />
+                @endisset
```

Comment:
As mentioned in the channel, you'll probably need to do the following here: 

```suggestion
            @isset($interpretationName)
                    @isset($interpretationNameSpace)
                        <x-interpretation class="interpretation--start" name="{{ __($interpretationName, [], 'en') }}"
                        namespace="{{ $interpretationNameSpace }}" />
                    @else
                        <x-interpretation class="interpretation--start" name="{{ __($interpretationName, [], 'en') }}" />
                    @endisset
                @endisset
```

This is because blade directives don't work in blade components. 

See: https://github.com/laravel/framework/issues/32045